src/prototype_room_external.js

```diff
@@ -154,6 +146,12 @@ Room.prototype.externalHandleHighwayRoom = function() {
       Memory.powerBanks = {};
     }
     if (target !== null) {
+      // Fix for W8S9
```

Comment:
I think this part could even be removed. IIRC this was about handling newbie walls. Which never worked properly.

Also fine to leave it in.